### PR TITLE
fix: fixes cli not finding codemods, but actually

### PR DESCRIPTION
**Pull request by madou**
**TODO**

- [ ] Fix local run of CLI (it doesn't find codemods anymore)

### CODE REVIEW

**Comment by pgmanutd** [comment]
@madou: Thanks for cleaning up the tests 😄. Just see if we can fix "local run of CLI (it doesn't find codemods anymore)". Looks good otherwise.

**Review by alexreardon — APPROVED** [review]

**Comment by madou** [comment]
@pgmanutd fixed

**Comment by pgmanutd** [comment]
> @pgmanutd fixed (the error is just me stopping the process)
> 
> image

So we have replaced `process.cwd()` with `app-root-path` 🤔. If we run it in any repo, will it refer to the repo path or path where cli is installed? I am confused after reading https://www.npmjs.com/package/app-root-path#edge-case-global-clis

**Comment by madou** [comment]
🤔 i think it might handle it. i'll run a nightly and test this before merging

**Comment by pgmanutd** [comment]
> 🤔 i think it might handle it. i'll run a nightly and test this before merging

No worries. Lets test this once before shipping.

**Review by pgmanutd — APPROVED** [review]

**Comment by madou** [comment]
doesn't work 😭 

```sh
➜  test-compiled npx @compiled/cli@0.5.4-7e02b8a404f6056123a7cdc6fb17314e6deb8c0b.0
npx: installed 285 in 10.916s
👷 Compiled cli 👷
✔ Select which preset would you like to run? 🤔 · codemods
No codemods available right now.
➜  test-compiled npx @compiled/cli@0.5.4-7e02b8a404f6056123a7cdc6fb17314e6deb8c0b.0 --version
npx: installed 285 in 6.874s
0.5.4-7e02b8a404f6056123a7cdc6fb17314e6deb8c0b.0
➜  test-compiled
```

**Comment by pgmanutd** [comment]
> doesn't work 😭
> 
> ```shell
> ➜  test-compiled npx @compiled/cli@0.5.4-7e02b8a404f6056123a7cdc6fb17314e6deb8c0b.0
> npx: installed 285 in 10.916s
> 👷 Compiled cli 👷
> ✔ Select which preset would you like to run? 🤔 · codemods
> No codemods available right now.
> ➜  test-compiled npx @compiled/cli@0.5.4-7e02b8a404f6056123a7cdc6fb17314e6deb8c0b.0 --version
> npx: installed 285 in 6.874s
> 0.5.4-7e02b8a404f6056123a7cdc6fb17314e6deb8c0b.0
> ➜  test-compiled
> ```

Yeah it looked like it wont work after I read https://www.npmjs.com/package/app-root-path#edge-case-global-clis
damn!!

**Comment by pgmanutd** [comment]
We can use dev vs prod or something. `app-root-path` for dev and `process.cwd()` for prod build.

**Comment by madou** [comment]
latest works!

```
➜  test-compiled npx @compiled/cli@0.5.4-6ff480436f67d8e51aeadc05b9481545fd71cee4.0
npx: installed 285 in 10.902s
👷 Compiled cli 👷
✔ Select which preset would you like to run? 🤔 · codemods
? Select which codemod would you like to run? 🤔 …
emotion-to-compiled
styled-components-to-compiled
```

File: packages/cli/src/presets/codemods/utils/transforms.tsx [hunk 1]
```diff
@@ -2,8 +2,15 @@ import path, { ParsedPath } from 'path';
 import glob from 'glob';
 import appRoot from 'app-root-path';
 
+/**
+ * Local run is defined as running in ts-node.
+ */
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+const isLocalRun = typeof process[Symbol.for('ts-node.register.instance')] === 'object';
```
Comment (pgmanutd) [inline packages/cli/src/presets/codemods/utils/transforms.tsx:10]: hehe 😛. Its the same one. It works 🎉

Comment (madou) [reply packages/cli/src/presets/codemods/utils/transforms.tsx:10]: yeah used ur hacks hehehe

**Review by pgmanutd — APPROVED** [review]
